satpy/tests/test_data_download.py

```diff
@@ -103,6 +111,13 @@ def _get_comp_find_conditions(comp_sensors, found_files):
     return comp_cond
 
 
+def _get_mod_find_conditions(comp_sensors, found_files):
```

Comment:
Could you find a better name for this function? I'm still not 100% what this returns...

Reply:
Another thought: wouldn't it be better to have this function do the assertion directly?

Reply:
Yeah, looking at this now I'm not sure why I didn't do it that way. Maybe for debugging or something. Not sure. Fixing now.